src/core/location/service/location.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable } from '@nestjs/common';
+import { HttpService } from '@nestjs/axios';
+import { envData } from 'src/config/datasource';
+
+@Injectable()
+export class LocationService {
+  constructor(private readonly httpService: HttpService) {}
+
+  async findAll(name?: string) {
```

Comment:
Recomiendo poner como diablos es la respuesta :B
```suggestion
  async findAll(name?: string): Promise<Location> {
```